types/relay-runtime/lib/store/RelayModernStore.d.ts

```diff
@@ -22,7 +21,11 @@ import {
 } from './RelayConnection';
 
 export class RelayModernStore implements Store {
-    constructor(source: MutableRecordSource, gcScheduler?: Scheduler, operationLoader?: OperationLoader | null);
+    constructor(source: MutableRecordSource, options?: {
+        gcScheduler?: Scheduler | null,
+        operationLoader?: OperationLoader | null
+        gcReleaseBufferSize?: number | null,
```

Comment:
Same here, `| undefined`, if I'm right

Reply:
👍 you are right

Reply:
Ah, so actually no - TS behaves differently:

```text
ERROR: 25:42  no-redundant-undefined  Property is optional, so no need to specify `undefined` as a possible value. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-redundant-undefined.md
```

Reply:
https://github.com/Microsoft/dtslint/blob/master/docs/no-redundant-undefined.md